app/build.gradle.kts

```diff
@@ -213,6 +213,7 @@ dependencies {
     androidTestImplementation(libs.androidx.work.testing)
     androidTestImplementation(libs.hilt.android.testing)
     androidTestImplementation(libs.junit)
+    androidTestImplementation(libs.kotlinx.coroutines.test)
```

Comment:
Is there a specific reason for using `runTest` instead of `runBlocking` in the tests? As I understand it, the main difference is skipping delays, and we don't need that.

With `runBlocking`, we could omit this dependency.